CHANGELOG.md

```diff
@@ -1,9 +1,32 @@
-# dbt_linkedin version.version
+# dbt_linkedin v0.11.0
+
+[PR #44](https://github.com/fivetran/dbt_linkedin/pull/44) includes the following updates:
+
+## Schema Changes
+**8 total changes • 0 possible breaking changes
+| Data Model                                    | Change Type | Old Name | New Name                                  | Notes                                                             |
+|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| [linkedin_ads__monthly_campaign_country_report](https://fivetran.github.io/dbt_linkedin/#!/model/model.linkedin.linkedin_ads__monthly_campaign_country_report)       | New transformation model   |          |  | Table that represents the monthly performance of a campaign at the country level.               |
+| [linkedin_ads__monthly_campaign_region_report]((https://fivetran.github.io/dbt_linkedin/#!/model/model.linkedin.linkedin_ads__monthly_campaign_region_report))       | New transformation model   |          |  | Table that represents the monthly performance of a campaign at the region level.              |
+| stg_tiktok_ads__geo_tmp       | New temp model   |          |  | Temp model added for `geo`.               |
```

Comment:
Same recommendation to update these models with links to the DAG.

Reply:
Updated